runtime/src/lib.rs

```diff
@@ -1031,4 +1059,31 @@ impl_runtime_apis! {
     }
 }
 
```

Comment:
do we want the frame_benchmarking https://github.com/substrate-developer-hub/substrate-parachain-template/blob/master/runtime/src/lib.rs#L590 ?

Reply:
I don't know why it was not included. frame_benchmarking was also present in `rococo-v1`.  @festelo 

We can do this as a part of https://github.com/DataHighway-DHX/node/issues/228